src/interface/src/app/top-bar/top-bar.component.html

```diff
@@ -51,28 +16,18 @@
       class="nav-link"
       data-id="feedback"
       target="_blank"
-      rel="noopener noreferrer"
-      *featureFlag="'new_navigation'">
+      rel="noopener noreferrer">
       Feedback
     </a>
 
     <!-- Account Name & Button -->
-    <a
-      mat-icon-button
-      routerLink="/help"
```

Comment:
This may be a larger question for Scott/Michele, but is there a way for a user to reach the user guide from the home page? Just realized it doesn't look like it's accessible until the Explore page

Reply:
I'll double check on the email thread about this